azure_li_services/network.py

```diff
@@ -0,0 +1,131 @@
+# Copyright (c) 2017 SUSE Linux GmbH.  All rights reserved.
+#
```

Comment:
It's May 2018, I know it is hard to accept that time marches on, but it does .....

Reply:
:) I'll change it